### DISCUSSION TITLE
求助：my2sql 没有找到匹配的记录

### DISCUSSION THREAD

**Discussion by LIN-HongTao** [hhyo/Archery#2111] [Q&A]
版本：1.9.1
部署方式：docker部署
binlog格式：row
配置：
![image](https://user-images.githubusercontent.com/22323046/229447096-0a9b7ded-d41b-4cac-b4f1-5b90d7af2563.png)

结果：
![image](https://user-images.githubusercontent.com/22323046/229447376-bad7954e-e416-42f6-beab-0f89d3054974.png)

**Comment by nick2wang** [comment]
手动执行一下my2sql命令看是否有结果

**Reply by LIN-HongTao** [reply]
![image](https://user-images.githubusercontent.com/22323046/229661677-a5d2d984-fd8c-4a34-b358-714f62042497.png)

**Reply by nick2wang** [reply]
根据前台条件执行一下
